server/src/main/java/com/google/musicanalysis/site/YoutubeServlet.java

```diff
@@ -2,7 +2,7 @@
 import com.google.musicanalysis.util.Secrets;
 import com.google.musicanalysis.util.URLEncodedBuilder;
 
-import com.google.gson.JsonParser;
+import com.google.gson.*;
```

Comment:
nit: I personally only include the files that I need within a package. 

https://stackoverflow.com/questions/14897448/use-of-in-import-statement-in-java

Reply:
got it! I added 3 more packages